lib/timeline.js

```diff
@@ -0,0 +1,15 @@
+const dates = require('../out/dates.json')
+
+const datetime = new Date()
+
+const competitionOpen = new Date(dates.competition_open_starts)
+
+const noClaims = new Date(dates.competition_open_ends)
+
+const totalTime = (noClaims - competitionOpen)/(24*3600*1000)
+
+const timeLeft = (noClaims-datetime)/(24*3600*1000)
+
+const daysLeft = parseInt(timeLeft)
+
+const percentagePassed = parseInt(100-(timeLeft/totalTime)*100)
```

Comment:
What's this file?

Reply:
I tried to modularize the process, now it's removed